src/App.js

```diff
@@ -1,17 +1,29 @@
+import React, { useEffect } from 'react';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 import LoginPage from './components/Pages/login/LoginPage';
 import MainPage from './components/Pages/main/MainPage';
+import ReservationForm from './components/Pages/reserve/Reserve';
+import { fetchDoctors } from './redux/doctors/DoctorList';
+
 import './App.css';
 
-const App = () => (
-  <div className="App">
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchDoctors());
+  }, []);
+
+  return (
+
     <Router>
       <Routes>
         <Route exact path="login" element={<LoginPage />} />
+        <Route path="/reserve" element={<ReservationForm />} />
```

Comment:
The reservation form is not supposed to be put here. We already created it's component in the rooting in the MainPage.js . I understand if some part are not clear enaugh, but you can reach out if you need any explanations.

Reply:
okay I will change my route